verification/curator-service/ui/src/components/Profile.tsx

```diff
@@ -24,11 +42,310 @@ const styles = makeStyles((theme: Theme) => ({
     },
 }));
 
+const useStyles = makeStyles((theme: Theme) => ({
+    checkboxRoot: {
+        display: 'block',
+    },
+    required: {
+        color: theme.palette.error.main,
+    },
+    inpputField: {
+        display: 'block',
+        width: '240px',
+        marginBottom: '10px',
+    },
+    signInButton: {
+        marginTop: '10px',
+        marginBottom: '10px',
+    },
+    checkboxLabel: {
+        fontSize: '14px',
+    },
+    link: {
+        fontWeight: 'bold',
+        color: theme.palette.primary.main,
+        cursor: 'pointer',
+    },
+    forgotPassword: {
+        fontWeight: 'normal',
+        color: theme.palette.primary.main,
+        cursor: 'pointer',
+        fontSize: 'small',
+        marginTop: '-8px',
+        display: 'flex',
+        justifyContent: 'flex-end',
+    },
+    labelRequired: {
+        color: theme.palette.error.main,
+    },
+    title: {
+        margin: '10px 0',
+        fontWeight: 700,
+    },
+    googleButton: {
+        // margin: '35px 0 0 0',
+        fontWeight: 400,
+    },
+    formFlexContainer: {
+        display: 'flex',
+        gap: '80px',
+        justifyContent: 'center',
+        marginTop: '30px',
+    },
+}));
+
+interface FormValues {
+    oldpassword: string;
+    password: string;
+    passwordConfirmation: string;
+}
+
+interface ChangePasswordFormInProfileProps {
+    disabled?: boolean;
+}
+
+export function ChangePasswordFormInProfile({
+    disabled,
+}: ChangePasswordFormInProfileProps): JSX.Element {
+    const classes = useStyles();
+    const dispatch = useAppDispatch();
+    const history = useHistory();
+
+    const passwordReset = useAppSelector(selectPasswordReset);
+    const [passwordVisible, setPasswordVisible] = useState(false);
+    const [
+        passwordConfirmationVisible,
+        setPasswordConfirmationVisible,
+    ] = useState(false);
+
+    // After successful password reset redirect user to landing page and show snackbar alert
+    useEffect(() => {
```

Comment:
This is not necessary here, after successful password change user should see snackbar with success message, but no redirect is needed